bin/upload_telemetry_schema.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python
```

Comment:
This used to be a SocorroApp, but I rewrote it as a cli using click and moved it to `bin/`.